explore/src/main/scala/explore/targeteditor/TargetSummaryTable.scala

```diff
@@ -174,94 +182,76 @@ object TargetSummaryTable {
                 )
               )
               .setDisableSortBy(true)
-          )).toJSArray
-
-      val rawData = props.pointingsWithObs.targets.toList.toJSArray
-
-      tableComponent(
-        TableComponentProps(TargetTable.Options(columns, rawData).setAutoResetSortBy(false),
-                            props.hiddenColumns,
-                            props.renderInTitle
-        )
-      )
-    }
-  }
-
-  protected val component =
-    ScalaComponent
-      .builder[Props]
-      .renderBackend[Backend]
-      .configure(Reusability.shouldComponentUpdate)
-      .build
-
-  protected final case class TableComponentProps(
-    options:       TargetTable.OptionsType,
-    hiddenColumns: View[Set[String]],
-    renderInTitle: Tile.RenderInTitle
-  ) extends ReactProps[TargetSummaryTable](TargetSummaryTable.component)
-
-  // Horrible hack while we don't fully have hooks.
-  // Reusability is handled in class component, instead of the need to useMemo.
-  // Table is only rerendered when needed, thus avoiding the loop in react-table when passing unstable columns or data.
-  protected val tableComponent =
-    ScalaFnComponent[TableComponentProps] { props =>
-      val tableInstance = TargetTable.use(
-        props.options.setInitialStateFull(
-          TargetTable
-            .State()
-            .setHiddenColumns(
-              props.hiddenColumns.get.toList
-                .map(col => col: IdType[TargetResult])
-                .toJSArray
-            )
-        )
+          )
+      }
+      .useMemoBy((props, _) => props.pointingsWithObs)((_, _) => _.targets.toList) // Memo rows
+      .useTableBy_(TargetTable)(
```

Comment:
What is the difference between `useTableBy()` and `useTableBy_()`?

Reply:
I see. With or without `modOpts`(now I just need to understand where that is coming from...). I wonder if we should make that clearer in the names?

Reply:
Ah-ha. I grok it now. 🕺

Reply:
>I wonder if we should make that clearer in the names?

Agreed, I'll do that.